scripts/model/weather.js

```diff
@@ -0,0 +1,28 @@
+var weatherAPI = {};
+
+$.get('http://api.wunderground.com/api/f25e47528444be94/conditions/q/OR/Portland.json')
+	.done(function(data){
+	console.log(data);
+	weatherAPI = data;
+	$(Weather).text(
```

Comment:
This is the view within the model section. See if you can separate out the MVC concerns a bit more.